File: frontend/models/product/productPreviewPagingVm.ts
import {ProductPreviewVm} from "@/models/product/ProductPreviewVm";

export type ProductPreviewPagingVm = {
    productPreviewsPayload: ProductPreviewVm[],
    pageIndex: number,
    pageSize: number,
    totalElements: number,
    totalPages: number,
    isLast : boolean


}

File: product/src/main/java/com/lexuancong/product/viewmodel/product/ProductPreviewPagingVm.java
package com.lexuancong.product.viewmodel.product;

import java.util.List;

public record ProductPreviewPagingVm(
        List<ProductPreviewVm> productPreviewsPayload,
        int pageIndex,
        int pageSize,
        int totalElements,
        int totalPages,
        boolean isLast
) {

}
